tests/vm_test.go

```diff
@@ -1326,61 +1323,59 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					By("Waiting for VM to be ready")
 					Eventually(ThisVM(vm), 360*time.Second, 1*time.Second).Should(BeReady())
 				})
-				DescribeTable("with a failing VMI and the kubevirt.io/keep-launcher-alive-after-failure annotation", func(keepLauncher string) {
-					// The estimated execution time of one test is 400 seconds.
-					By("Creating a Kernel Boot VMI with a mismatched disk")
-					vmi := utils.GetVMIKernelBootWithRandName()
-					vmi.Spec.Domain.Firmware.KernelBoot.Container.Image = cd.ContainerDiskFor(cd.ContainerDiskCirros)
 
-					By("Creating a VM with RunStrategyManual")
-					vm := libvmi.NewVirtualMachine(vmi)
+				DescribeTable("with a failing VMI and the kubevirt.io/keep-launcher-alive-after-failure annotation", func(keepLauncher string) {
+					By("creating a Running VM")
+					vm := libvmi.NewVirtualMachine(libvmifact.NewCirros(
+						libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+						libvmi.WithNetwork(v1.DefaultPodNetwork()),
+						libvmi.WithAnnotation(v1.KeepLauncherAfterFailureAnnotation, keepLauncher),
+						libvmi.WithAnnotation(v1.FuncTestLauncherFailFastAnnotation, ""),
+					), libvmi.WithRunStrategy(v1.RunStrategyOnce))
```

Comment:
This is a bit strange: we have two options here, one of them with many other options. Is there a reason for that?

Can we move the `libvmi.WithRunStrategy(v1.RunStrategyOnce)` option to be a parameter of the `NewVirtualMachine` option?

Reply:
The reason for the different options is that one set of options is for the VM and another for the VMI. The VMI doesn't have a runStrategy field in its spec; therefore, we need both options.